interface/templates/interface/submission_list.html

```diff
@@ -15,12 +15,12 @@
 
     <body>
 
-        <div class="container">
+        <div class="container mt-2">
             <div class="row">
-                <a class="btn btn-primary" href='{{ homepage_url }}'>Homepage</a>
+                <a class="btn btn-primary mr-2" href='{{ homepage_url }}'>Homepage</a>
                 <a class="btn btn-primary" href='{{ logout_url }}'>Logout</a>
             </div>
```

Comment:
Since you're changing these, let's use a bootstrap navbar, it's a better fit for the purpose:

https://getbootstrap.com/docs/4.1/components/navbar/

And you can extract it into a reusable template.